singolo/js/script.js

```diff
@@ -0,0 +1,253 @@
+const navigation = document.querySelector(".navigation");
```

Comment:
similar variables will be confusing _navigation_ and _header__navigation_ and _nav_